pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go

```diff
@@ -661,7 +674,10 @@ func getNodeWithPodAndVolumeLimits(limitSource string, pods []*v1.Pod, limit int
 	default:
 		// Do nothing.
 	}
-
-	nodeInfo.SetNode(node)
+	if isNilNode {
```

Comment:
We want `csiNode` to be nil, not `node`

Reply:
Then we should modify the getFakeCSIPVLister so that when we call Get the csiNode is nil.